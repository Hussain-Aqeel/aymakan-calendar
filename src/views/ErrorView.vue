<template>

  <head>
    <title>Not Found</title>
  </head>

  <div class="container flex flex-col items-center mx-auto">
    <h1 id="title" class="text-2xl font-bold">Error 404</h1>
    <img src="/images/not-found.png"
         alt="error"
         class="w-72 h-96" />
    <p class="text-xl">This Page is Lost in Space</p>
    <router-link to="/" class="px-5 py-3 mt-8 text-xl font-bold text-black duration-300 bg-amber-400 rounded-xl hover:bg-amber-300">
      Go Home
    </router-link>
  </div>

</template>

<script>
export default {

}
</script>

<style scoped>
  @import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap');
  @import url('https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap');

  body {
    font-family: 'Nunito', sans-serif;
  }

  #title {
    font-family: 'Press Start 2P', cursive;
  }

</style>