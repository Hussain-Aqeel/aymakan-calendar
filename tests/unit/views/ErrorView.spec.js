import { shallowMount } from '@vue/test-utils';
import ErrorView from '@/views/ErrorView.vue';

describe('ErrorView.vue', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(ErrorView)
  })

  test('snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})