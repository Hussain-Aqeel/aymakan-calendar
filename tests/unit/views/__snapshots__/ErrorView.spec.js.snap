// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorView.vue snapshot 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      
      <head>
        <title>
          Not Found
        </title>
      </head>
      <div
        class="container mx-auto flex flex-col items-center"
      >
        <h1
          class="text-2xl font-bold"
          id="title"
        >
          Error 404
        </h1>
        <img
          alt="error"
          class="w-72 h-96"
          src="/images/not-found.png"
        />
        <p
          class="text-xl"
        >
          This Page is Lost in Space
        </p>
        <router-link
          class="py-3 px-5 mt-8 bg-amber-400 text-black text-xl font-bold rounded-xl hover:bg-amber-300 duration-300"
          to="/"
        >
           Go Home 
        </router-link>
      </div>
      
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {
        "transition": Object {
          "name": "transition",
        },
        "transition-group": Object {
          "name": "transition-group",
        },
      },
      "config": Object {
        "compilerOptions": Object {},
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "emitsCache": WeakMap {},
      "mixins": Array [
        Object {
          "beforeCreate": [Function],
        },
      ],
      "optionsCache": WeakMap {},
      "propsCache": WeakMap {},
      "provides": Object {},
      "reload": [Function],
    },
    "_instance": Object {
      "a": null,
      "accessCache": Object {},
      "appContext": Object {
        "app": [Circular],
        "components": Object {
          "transition": Object {
            "name": "transition",
          },
          "transition-group": Object {
            "name": "transition-group",
          },
        },
        "config": Object {
          "compilerOptions": Object {},
          "errorHandler": undefined,
          "globalProperties": Object {},
          "isNativeTag": [Function],
          "optionMergeStrategies": Object {},
          "performance": false,
          "warnHandler": undefined,
        },
        "directives": Object {},
        "emitsCache": WeakMap {},
        "mixins": Array [
          Object {
            "beforeCreate": [Function],
          },
        ],
        "optionsCache": WeakMap {},
        "propsCache": WeakMap {},
        "provides": Object {},
        "reload": [Function],
      },
      "asyncDep": null,
      "asyncResolved": false,
      "attrs": Object {},
      "bc": null,
      "bm": null,
      "bu": null,
      "bum": null,
      "c": null,
      "components": null,
      "ctx": Object {},
      "da": null,
      "data": Object {},
      "directives": null,
      "ec": null,
      "effect": ReactiveEffect {
        "active": true,
        "allowRecurse": true,
        "deps": Array [
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
        ],
        "fn": [Function],
        "onTrack": undefined,
        "onTrigger": undefined,
        "parent": undefined,
        "scheduler": [Function],
      },
      "emit": [Function],
      "emitsOptions": null,
      "emitted": null,
      "exposeProxy": null,
      "exposed": null,
      "inheritAttrs": undefined,
      "isDeactivated": false,
      "isMounted": true,
      "isUnmounted": false,
      "m": null,
      "next": null,
      "parent": null,
      "props": Object {},
      "propsDefaults": Object {},
      "propsOptions": Array [
        Object {},
        Array [],
      ],
      "provides": Object {},
      "proxy": Object {},
      "refs": Object {
        "VTU_COMPONENT": Object {
          "hasOwnProperty": [Function],
        },
      },
      "render": [Function],
      "renderCache": Array [],
      "root": [Circular],
      "rtc": null,
      "rtg": null,
      "scope": EffectScope {
        "active": true,
        "cleanups": Array [],
        "effects": Array [
          ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": Array [
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
            ],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
        ],
      },
      "setupContext": null,
      "setupState": Object {},
      "slots": Object {},
      "sp": null,
      "subTree": Object {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": null,
        "children": null,
        "component": Object {
          "a": null,
          "accessCache": Object {
            "@@__IMMUTABLE_ITERABLE__@@": 0,
            "@@__IMMUTABLE_RECORD__@@": 0,
            "_isMockFunction": 0,
            "constructor": 0,
            "hasOwnProperty": 4,
            "isVueInstance": 0,
            "toJSON": 0,
            Symbol(Symbol.toStringTag): 0,
          },
          "appContext": Object {
            "app": [Circular],
            "components": Object {
              "transition": Object {
                "name": "transition",
              },
              "transition-group": Object {
                "name": "transition-group",
              },
            },
            "config": Object {
              "compilerOptions": Object {},
              "errorHandler": undefined,
              "globalProperties": Object {},
              "isNativeTag": [Function],
              "optionMergeStrategies": Object {},
              "performance": false,
              "warnHandler": undefined,
            },
            "directives": Object {},
            "emitsCache": WeakMap {},
            "mixins": Array [
              Object {
                "beforeCreate": [Function],
              },
            ],
            "optionsCache": WeakMap {},
            "propsCache": WeakMap {},
            "provides": Object {},
            "reload": [Function],
          },
          "asyncDep": null,
          "asyncResolved": false,
          "attrs": Object {},
          "bc": null,
          "bm": null,
          "bu": null,
          "bum": null,
          "c": null,
          "components": Object {},
          "ctx": Object {
            "hasOwnProperty": [Function],
          },
          "da": null,
          "data": Object {},
          "directives": null,
          "ec": null,
          "effect": ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": Array [],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
          "emit": [Function],
          "emitsOptions": null,
          "emitted": null,
          "exposeProxy": null,
          "exposed": null,
          "inheritAttrs": undefined,
          "isDeactivated": false,
          "isMounted": true,
          "isUnmounted": false,
          "m": null,
          "next": null,
          "parent": [Circular],
          "props": Object {},
          "propsDefaults": Object {},
          "propsOptions": Array [
            Object {},
            Array [],
          ],
          "provides": Object {},
          "proxy": Object {
            "hasOwnProperty": [Function],
          },
          "refs": Object {},
          "render": [Function],
          "renderCache": Array [],
          "root": [Circular],
          "rtc": null,
          "rtg": null,
          "scope": EffectScope {
            "active": true,
            "cleanups": Array [],
            "effects": Array [
              ReactiveEffect {
                "active": true,
                "allowRecurse": true,
                "deps": Array [],
                "fn": [Function],
                "onTrack": undefined,
                "onTrigger": undefined,
                "parent": undefined,
                "scheduler": [Function],
              },
            ],
          },
          "setupContext": null,
          "setupState": Object {},
          "slots": Object {},
          "sp": null,
          "subTree": Object {
            "__v_isVNode": true,
            "__v_skip": true,
            "anchor": ,
            "appContext": null,
            "children": Array [
              Object {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": Array [
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "Not Found",
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <title>
                      Not Found
                    </title>,
                    "key": null,
                    "patchFlag": 0,
                    "props": null,
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 9,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "title",
                  },
                ],
                "component": null,
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <head>
                  <title>
                    Not Found
                  </title>
                </head>,
                "key": null,
                "patchFlag": -1,
                "props": null,
                "ref": null,
                "scopeId": null,
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "head",
              },
              Object {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": Array [
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "Error 404",
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <h1
                      class="text-2xl font-bold"
                      id="title"
                    >
                      Error 404
                    </h1>,
                    "key": null,
                    "patchFlag": -1,
                    "props": Object {
                      "class": "text-2xl font-bold",
                      "id": "title",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 9,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "h1",
                  },
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": null,
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <img
                      alt="error"
                      class="w-72 h-96"
                      src="/images/not-found.png"
                    />,
                    "key": null,
                    "patchFlag": -1,
                    "props": Object {
                      "alt": "error",
                      "class": "w-72 h-96",
                      "src": "/images/not-found.png",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 1,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "img",
                  },
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "This Page is Lost in Space",
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <p
                      class="text-xl"
                    >
                      This Page is Lost in Space
                    </p>,
                    "key": null,
                    "patchFlag": -1,
                    "props": Object {
                      "class": "text-xl",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 9,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "p",
                  },
                  Object {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": Array [
                      Object {
                        "__v_isVNode": true,
                        "__v_skip": true,
                        "anchor": null,
                        "appContext": null,
                        "children": " Go Home ",
                        "component": null,
                        "dirs": null,
                        "dynamicChildren": null,
                        "dynamicProps": null,
                        "el":  Go Home ,
                        "key": null,
                        "patchFlag": 0,
                        "props": null,
                        "ref": null,
                        "scopeId": null,
                        "shapeFlag": 8,
                        "slotScopeIds": null,
                        "ssContent": null,
                        "ssFallback": null,
                        "staticCount": 0,
                        "suspense": null,
                        "target": null,
                        "targetAnchor": null,
                        "transition": null,
                        "type": Symbol(Text),
                      },
                    ],
                    "component": null,
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <router-link
                      class="py-3 px-5 mt-8 bg-amber-400 text-black text-xl font-bold rounded-xl hover:bg-amber-300 duration-300"
                      to="/"
                    >
                       Go Home 
                    </router-link>,
                    "key": null,
                    "patchFlag": 0,
                    "props": Object {
                      "class": "py-3 px-5 mt-8 bg-amber-400 text-black text-xl font-bold rounded-xl hover:bg-amber-300 duration-300",
                      "to": "/",
                    },
                    "ref": null,
                    "scopeId": null,
                    "shapeFlag": 17,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "router-link",
                  },
                ],
                "component": null,
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <div
                  class="container mx-auto flex flex-col items-center"
                >
                  <h1
                    class="text-2xl font-bold"
                    id="title"
                  >
                    Error 404
                  </h1>
                  <img
                    alt="error"
                    class="w-72 h-96"
                    src="/images/not-found.png"
                  />
                  <p
                    class="text-xl"
                  >
                    This Page is Lost in Space
                  </p>
                  <router-link
                    class="py-3 px-5 mt-8 bg-amber-400 text-black text-xl font-bold rounded-xl hover:bg-amber-300 duration-300"
                    to="/"
                  >
                     Go Home 
                  </router-link>
                </div>,
                "key": null,
                "patchFlag": 0,
                "props": Object {
                  "class": "container mx-auto flex flex-col items-center",
                },
                "ref": null,
                "scopeId": null,
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "div",
              },
            ],
            "component": null,
            "dirs": null,
            "dynamicChildren": Array [],
            "dynamicProps": null,
            "el": ,
            "key": null,
            "patchFlag": 64,
            "props": null,
            "ref": null,
            "scopeId": null,
            "shapeFlag": 16,
            "slotScopeIds": null,
            "ssContent": null,
            "ssFallback": null,
            "staticCount": 0,
            "suspense": null,
            "target": null,
            "targetAnchor": null,
            "transition": null,
            "type": Symbol(Fragment),
          },
          "suspense": null,
          "suspenseId": 0,
          "type": Object {
            "components": Object {},
            "render": [Function],
          },
          "u": null,
          "uid": 1,
          "um": null,
          "update": [Function],
          "vnode": [Circular],
          "withProxy": null,
        },
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": ,
        "key": null,
        "patchFlag": 0,
        "props": Object {
          "ref": "VTU_COMPONENT",
        },
        "ref": Object {
          "f": false,
          "i": [Circular],
          "k": undefined,
          "r": "VTU_COMPONENT",
        },
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": Object {
          "components": Object {},
          "render": [Function],
        },
      },
      "suspense": null,
      "suspenseId": 0,
      "type": Object {
        "name": "VTU_ROOT",
        "render": [Function],
      },
      "u": null,
      "uid": 0,
      "um": null,
      "update": [Function],
      "vnode": Object {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": Object {
          "app": [Circular],
          "components": Object {
            "transition": Object {
              "name": "transition",
            },
            "transition-group": Object {
              "name": "transition-group",
            },
          },
          "config": Object {
            "compilerOptions": Object {},
            "errorHandler": undefined,
            "globalProperties": Object {},
            "isNativeTag": [Function],
            "optionMergeStrategies": Object {},
            "performance": false,
            "warnHandler": undefined,
          },
          "directives": Object {},
          "emitsCache": WeakMap {},
          "mixins": Array [
            Object {
              "beforeCreate": [Function],
            },
          ],
          "optionsCache": WeakMap {},
          "propsCache": WeakMap {},
          "provides": Object {},
          "reload": [Function],
        },
        "children": null,
        "component": [Circular],
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": ,
        "key": null,
        "patchFlag": 0,
        "props": null,
        "ref": null,
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": Object {
          "name": "VTU_ROOT",
          "render": [Function],
        },
      },
      "withProxy": null,
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "compilerOptions": Object {},
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.2.37",
  },
  "__setProps": [Function],
  "componentVM": Object {
    "hasOwnProperty": [Function],
  },
  "isDisabled": [Function],
  "rootVM": Object {},
  "wrapperElement": ,
}
`;
